### ISSUE TITLE
Alert callback never invoked

### ISSUE THREAD

**Issue by kohanyirobert** [pkkid/python-plexapi#353]
I have a code that looks something like this.

```
def cb(data):
    print(data)

server = PlexServer(url, token)
library = server.library
section = library.section(section_title)
notifier = server.startAlertListener(callback=cb)

section.update()
```

I can my library being updated in Plex but the callback function never called. The thread that's returned by `startAlertListener` is started (if I call `start()` on it myself I get an error).

I've tired with ` a13d17c` and with `3.0.6` as well, neither worked.
I've installed `0.56.0` of `websocket-client`.

Any idea?

**Comment by Hellowlol** [comment]
notifier is a thread. It dont work as the program finishes. You need to do notifier.join or use a while loop.

**Comment by kohanyirobert** [comment]
I've updated my code to avoid confusion. I do use a `while` loop and all, please see the updated question.

**Comment by Hellowlol** [comment]
Your example will not work since your killing the notifier. You can try something like this:

````from sys import argv
from time import sleep
from plexapi.server import PlexServer

url = argv[1]
token = argv[2]
section_title = argv[3]

def cb(data):
    print('This zhe data', data)
    

server = PlexServer(url, token)
library = server.library
section = library.section(section_title)
notifier = server.startAlertListener(callback=cb)

if section.refreshing:
    print('already refrehsing')
else:
    print('start refreshing')
    section.update()

while True:
    try:
        print('notifier sleeping')
        sleep(1)
    except KeyboardInterrupt:
        break

notifier.stop()

print('ok')
````

**Comment by Hellowlol** [comment]
I noticed that i have pinned websocket in another project im using: websocket-client==0.47, but honestly i would use another library for websocket. I have thought about using https://github.com/wildfoundry/dataplicity-lomond i like the auto reconnect it has :)

**Comment by andrey-yantsen** [comment]
The issue we had with `websocket-client` was resolved in v0.53, so current versions should work without any problems

**Comment by kohanyirobert** [comment]
Sorry for the late reply.

> Your example will not work since your killing the notifier. You can try something like this:

I'm _trying_ to kill the notifier on purpose, but it never will be killed, because the `cb` callback method never fires, that's my problem. `print('This zhe data', data)` never runs.

> I noticed that i have pinned websocket in another project im using: websocket-client==0.47

Pinning the version to 0.47 did the trick, not sure about the underlying issue. I've tried 0.53 and the latest version as well, neither worked.
